core/src/test/java/com/linecorp/armeria/internal/server/annotation/OrElseResponseConverterFunctionTest.java

```diff
@@ -0,0 +1,74 @@
+package com.linecorp.armeria.internal.server.annotation;
```

Comment:
Could you add a copyright?

Reply:
So fast! You reviewed it before I ran the build which reminded me too 😁 I'll add it

Reply:
Addressed in 39c230de